models/toolbox.py

```diff
@@ -7,11 +7,11 @@
 import os
 
 
-DEFAULT_TOOLBOX_DIR = os.path.join(
+DEFAULT_COMMAND_DIR = os.path.join(
     os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))),
```

Comment:
Ik denk dat het dit ook kan zijn:
```suggestion
    os.path.dirname(os.path.abspath(__file__)),
```